digitalocean/blog.yaml

```diff
@@ -0,0 +1,46 @@
+# https://www.digitalocean.com/docs/app-platform/references/app-specification-reference/
+domains:
+- domain: blog.whatwg.org
+  type: PRIMARY
+  zone: whatwg.org
+name: blog
+region: nyc
+services:
+- dockerfile_path: Dockerfile
+  envs:
+  - key: WORDPRESS_DB_HOST
+    scope: RUN_TIME
+    value: ${db-mysql-nyc3-38716.HOSTNAME}:${db-mysql-nyc3-38716.PORT}
+  - key: WORDPRESS_DB_NAME
+    scope: RUN_TIME
+    value: ${db-mysql-nyc3-38716.DATABASE}
+  - key: WORDPRESS_DB_USER
+    scope: RUN_TIME
+    value: ${db-mysql-nyc3-38716.USERNAME}
+  - key: WORDPRESS_DB_PASSWORD
+    scope: RUN_TIME
+    value: ${db-mysql-nyc3-38716.PASSWORD}
+  - key: WORDPRESS_EXTRA_CONFIG
+    scope: RUN_TIME
+    value: |
+      define('DB_CHARSET', 'utf8mb4');
+  github:
+    branch: do-apps # TODO update
+    deploy_on_push: true
+    repo: whatwg/blog.whatwg.org
+  health_check:
+    http_path: /
+  http_port: 80
+  instance_count: 1
+  instance_size_slug: basic-xxs
+  name: blog
+  routes:
+  - path: /
+databases:
+- cluster_name: db-mysql-nyc3-38716
```

Comment:
Can we pick this name ourselves? If so, something more catchy?

Reply:
I tried and it kept telling me my name was invalid!! I can try to see if there's a rename option or something...